spec

```diff
@@ -604,7 +604,7 @@ beginseg
     align 0x40
```

Comment:
with these changes, is this spec align directive still necessary? Same with the `pad_text` above audio (that one is pretty ugly).

Reply:
The spec alignment is still required for IDO, the ALIGNEDx directives are only understood by gcc unfortunately

Reply:
(Disregard, I missed that these changes do not affect IDO)